src/h5web/dataset-visualizer/DimensionMapper.module.css

```diff
@@ -16,12 +16,12 @@
   display: flex;
 }
 
-.buttonGroupLabel {
+.dimensionName {
+  font-family: serif;
   margin-right: 0.5rem;
 }
 
-.btn {
-  composes: btn-clean from global;
+.btnAspect {
```

Comment:
A CSS class to give the button aspect without being a button !

Reply:
Ooh nice 👍